JamGame/Assets/Scripts/TileUnion/Tile/TileImpl.cs

```diff
@@ -73,13 +76,33 @@ public partial class TileImpl : MonoBehaviour
         }
 
         private readonly float selectLiftingHeight = 3;
+
+        [ReadOnly]
+        [SerializeField]
         private float unselectedYPosition;
+
+        [ReadOnly]
+        [SerializeField]
         private float selectedYPosition;
 
+        [ReadOnly]
+        [SerializeField]
+        private float unselectedFoundationYPosition;
+
+        [ReadOnly]
+        [SerializeField]
```

Comment:
```suggestion
        private float unselectedYPosition;
        
        private float selectedYPosition;
        
        private float unselectedFoundationYPosition;
```
Don't show it in the editor. It's already overloaded with information